app/src/main/java/org/oppia/android/app/settings/profile/DeleteProfileSuccessDialogFragment.kt

```diff
@@ -11,18 +11,16 @@ import org.oppia.android.app.administratorcontrols.AdministratorControlsActivity
 import org.oppia.android.app.fragment.FragmentComponentImpl
 import org.oppia.android.app.fragment.InjectableDialogFragment
 
-/** [DialogFragment] that notifies user after profile is successfully deleted. */
+/** [DialogFragment] that notifies the user after a profile is successfully deleted. */
 class DeleteProfileSuccessDialogFragment : InjectableDialogFragment() {
 
   companion object {
     const val TAG = "DELETE_SUCCESS_DIALOG_FRAGMENT"
 
     /**
-     * This function is responsible for displaying content in DialogFragment.
-     *
-     * @return [DeleteProfileSuccessDialogFragment]: DialogFragment
+     * This function returns a new instance of [DeleteProfileSuccessDialogFragment]
```

Comment:
This should actually be the exact wording I suggest, i.e.:

```kotlin
/** Returns a new instance of [DeleteProfileSuccessDialogFragment]. */
```

Reply:
Updated this

Reply:
This doesn't seem updated in the latest version of the PR. Did you push your changes?

Reply:
Updated now

Reply:
This still seems unchanged--I see the old documentation wording in the latest changes of the PR.